bots/checkout/src/main/java/org/openjdk/skara/bots/checkout/CheckoutBot.java

```diff
@@ -64,10 +64,7 @@ private static String urlEncode(URI uri) {
     }
 
     private URI webURI() {
-        var webURI = from.webUrl().toString();
-        if (!webURI.endsWith(".git")) {
-            webURI += ".git";
-        }
+        var webURI = from.remoteUrl().toString();
```

Comment:
Converting from URI to String and then back to URI doesn't seem like a good idea. This method is called in one place, so should probably just get rid of it.

Reply:
Oh, I didn't find this redundant operation.